_translator-files/po/zh-CN/Unregistered-Servers.po

```diff
@@ -1,146 +1,147 @@
 
 #. type: Plain text
 #: ../wiki/en/Unregistered-Servers.md:9
 msgid "{% include breadcrumb.html root=\"More\" branch1=\"Server Administration\" branch1-url=\"Running-a-Server\" %}"
-msgstr ""
+msgstr "{% include breadcrumb.html root=\"更多\" branch1=\"Server Administration\" branch1-url=\"Running-a-Server\" %}"
 
 #. type: Plain text
 #: ../wiki/en/Unregistered-Servers.md:13
 msgid "It is highly recommended to test your Server by registering it on one of the built-in Directories **first**. This will help you tackle general problems before you try unregistered mode."
-msgstr ""
+msgstr "强烈建议您先通过在其中一个内置目录中注册您的服务器来对其进行测试。这将帮助您在尝试未注册模式之前解决一般问题。"
 
 #. type: Title ##
 #: ../wiki/en/Unregistered-Servers.md:14
 #, no-wrap
 msgid "Setting up a Server behind a home router"
-msgstr ""
+msgstr "在家用路由器后面设置服务器"
 
 #. type: Plain text
 #: ../wiki/en/Unregistered-Servers.md:17
 msgid "If you set up your Server at home, you will probably need to change some settings in your router/firewall:"
-msgstr ""
+msgstr "如果您在家里设置服务器，您可能需要更改路由器/防火墙中的一些设置："
 
 #. type: Title ###
 #: ../wiki/en/Unregistered-Servers.md:18
 #, no-wrap
 msgid "Port forwarding"
-msgstr ""
+msgstr "端口转发"
 
 #. type: Plain text
 #: ../wiki/en/Unregistered-Servers.md:21
 msgid "Normally, people from outside your home network cannot see things inside it. To let external Jamulus Clients connect to your Server, you need to set up port forwarding in your router's settings. The exact setup differs for every router. For help, see your router's documentation or [portforward.com](https://portforward.com)."
-msgstr ""
+msgstr "通常，您家庭网络之外的人无法看到网络内部的情况。要让外部的 Jamulus 客户端连接到您的服务器，您需要在路由器的设置中设置端口转发。每种路由器的确切设置都不同。如需帮助，请参阅您的路由器文档或 [portforward.com](https://portforward.com) 。"
 
 #. type: Plain text
 #: ../wiki/en/Unregistered-Servers.md:23
 #, no-wrap
 msgid "**Note:** The default port for Jamulus is **UDP** (not TCP) port **22124**. You will usually forward the port **22124** from outside your network to the port **22124** of the computer running the Server.\n"
-msgstr ""
+msgstr "**注意:** Jamulus 的默认端口是 **UDP**（不是 TCP）端口 **22124**。您通常会将端口 **22124** 从网络外部转发到运行服务器的计算机的端口 **22124**。\n"
 
 #. type: Title ###
 #: ../wiki/en/Unregistered-Servers.md:24
 #, no-wrap
 msgid "Getting the public IP"
-msgstr ""
+msgstr "获取公网 IP"
 
 #. type: Plain text
 #: ../wiki/en/Unregistered-Servers.md:28
 msgid "To allow others to connect to your Server from the internet, give them your public IP address. You can [learn your current public IP address using Google](https://www.google.com/search?q=what+is+my+ip).  You should connect yourself using the local network address of the computer the Server is running on. If you are running a Client on the same computer as your Server, the address is `localhost` or `127.0.0.1`."
-msgstr ""
+msgstr "要允许其他人从互联网连接到您的服务器，请向他们提供您的公共 IP 地址。您可以[使用 Google 了解您当前的公共 IP 地址](https://www.google.com/search?q=what+is+my+ip)。您应该使用运行服务器的计算机的本地网络地址连接自己。如果您在与服务器相同的计算机上运行客户端，则地址为“localhost”或“127.0.0.1”。"
```

Comment:
Maybe it would be better to link some website which is guaranteed available in mainland china. I don't think google is.